spec/client_spec.js

```diff
@@ -275,22 +275,50 @@ describe('Client', function() {
         requestsCount++;
       });
 
-      it('returns a promise', function() {
+      it('returns a promise', function(done) {
         promise = subject.get('ticket.subject');
 
         expect(promise).to.be.a.promise;
+        expect(promise).to.eventually.become('test').and.notify(done);
```

Comment:
protip: if you don't want to have to notify done all the time, you can just return the promise to chai and it'll handle it.